src/loc_map/src/kalman_filter/observation_models.cpp

```diff
@@ -41,13 +41,73 @@ Eigen::Vector2f ObservationModel::observation_model(const Eigen::VectorXf &expec
 Eigen::MatrixXf ObservationModel::get_state_to_observation_matrix(
     const Eigen::VectorXf &expected_state, const unsigned int landmark_index,
     const unsigned int state_size) const {
+
+  // //direct insertion
+  // Eigen::MatrixXf validation_jacobian = Eigen::MatrixXf::Zero(2, state_size);
+
+  // validation_jacobian(0, 0) = -cos(-expected_state(2));
+  // validation_jacobian(1, 0) = -sin(-expected_state(2));
+  // validation_jacobian(0, 1) = sin(-expected_state(2));
+  // validation_jacobian(1, 1) = -cos(-expected_state(2));
+  // validation_jacobian(0, 2) = -expected_state(landmark_index) * sin(expected_state(2)) +
```

Comment:
What is this?

Reply:
directly inserts values into the matrix instead of calculating through multiplication

Reply:
wym, when would that be used?